versions-maven-plugin/src/it/it-dependency-updates-report-issue-1191/verify.groovy

```diff
@@ -0,0 +1,3 @@
+assert new File( basedir, "target/dependency-updates-report.xml" ).text
+        .replaceAll( '\\s*', '' )
+        .contains( '<artifactId>dummy-api</artifactId>' )
```

Comment:
for xml we can use `XmlSlurper`

Reply:
Ok, so while doing this I discovered that I was using XmlSlurper's find wrong. So corrected that too.